### PR TITLE
[Australia]: Setting up conf website 2024

### PR DESCRIPTION
**Pull request by swapnilogale**
----
📚 Documentation preview 📚: https://writethedocs-www--2155.org.readthedocs.build/

### CODE REVIEW

**Comment by plaindocs** [comment]
Loop me in on this whenever you need to mate.

**Comment by swapnilogale** [comment]
> Loop me in on this whenever you need to mate.

Just waiting on @mxsasha to set up the ticketing site and we should be good to review this. Also, we need to get shiham-jamaldeen@ shadowing this - how do we add him to the PR?

**Comment by plaindocs** [comment]
@swapnilogale figure out his GitHub username and @ him? :-D

**Comment by swapnilogale** [comment]
@plaindocs - This is ready for review. I'll double-check the ticket links to Tito and add a new post around CFP and tickets later today.

File: docs/_data/australia-2024-config.yaml [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 
-shortcode: australia # prague australia?
+shortcode: australia
 year: 2024
 city: Melbourne
 local_area: Australia
 area: Australia
 area_adj: Australian
 tz: AEDT
+tz_header: AEDT
```
Comment (plaindocs) [inline docs/_data/australia-2024-config.yaml]: huh?

File: docs/_data/australia-2024-config.yaml [hunk 2]
```diff
@@ -140,17 +136,17 @@ about:
...
   projector_ratio: "16:9"
   job_fair_room: "Sponsor area"
 
 cfp:
-  url: https://pretalx.com/wtd-australia-2023/cfp # Open July 15, 2023, 8 weeks to submit
-  ends: "September 22, 2023"           # 2 weeks to review and get confirmations
-  notification: "September 30, 2023"   # 8 weeks to write and prepare
+  url: https://pretalx.com/wtd-australia-2024/cfp # Open June 17, 2024, 10 weeks to submit
+  ends: "August 30, 2023"              # 2 weeks to review and get confirmations
+  notification: "September 13, 2023"   # 10 weeks to write and prepare
```
Comment (plaindocs) [inline docs/_data/australia-2024-config.yaml]: ending August feels super early for a December conf? 

Also, 2024!!

File: docs/_data/australia-2024-config.yaml [hunk 3]
```diff
@@ -160,9 +156,9 @@ unconf:
 
 
 grants:
-  url: https://forms.gle/Pi7zm72XPjebNzS16
-  ends: "October 15, 2023"
-  notification: "October 31, 2023"
+  url: https://forms.gle/va1aFysrH3N83jGk9
+  ends: "October 4, 2024"
+  notification: "October 11, 2024"
```
Comment (plaindocs) [inline docs/_data/australia-2024-config.yaml]: Also see #2161

File: docs/conf/australia/2024/cfp-email-templates.rst [hunk 4]
```diff
@@ -0,0 +1,323 @@
+:template: {{year}}/generic.html
+:orphan:
+
+Speaker email templates
+=======================
```
Comment (plaindocs) [inline docs/conf/australia/2024/cfp-email-templates.rst:5]: Are you using these btw?

File: docs/conf/australia/2024/code-of-conduct.rst [hunk 5]
```diff
@@ -0,0 +1,4 @@
+:template: {{year}}/generic.html
+
+.. include:: ../../../code-of-conduct.rst
+   :end-before: Portland conference Code of Conduct Team
```
Comment (plaindocs) [inline docs/conf/australia/2024/code-of-conduct.rst]: TIL

File: docs/conf/australia/2024/news/announcing-cfp-tickets.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
+:template: {{year}}/generic.html
+
+.. post:: June 15, 2024
```
Comment (plaindocs) [inline docs/conf/australia/2024/news/announcing-cfp-tickets.rst]: s/15/19/ I guess

**Review by plaindocs — APPROVED** [review]
Look good to me. Few queries, and I didn't go over with the MOST detailed comb.

File: docs/_data/australia-2024-config.yaml (hunk 2 above)
Comment (swapnilogale) [reply docs/_data/australia-2024-config.yaml]: Conference late Nov, so moved it up by a week :)

File: docs/_data/australia-2024-config.yaml (hunk 3 above)
Comment (swapnilogale) [reply docs/_data/australia-2024-config.yaml]: I thought that was merged - my bad. Will merge it in before pushing these changes.

File: docs/conf/australia/2024/cfp-email-templates.rst (hunk 4 above)
Comment (swapnilogale) [reply docs/conf/australia/2024/cfp-email-templates.rst:5]: No, but the build failed when I didn't have a bunch of these files, so added them in.

File: docs/conf/australia/2024/code-of-conduct.rst (hunk 5 above)
Comment (swapnilogale) [reply docs/conf/australia/2024/code-of-conduct.rst]: Updated

**Pull request merged**